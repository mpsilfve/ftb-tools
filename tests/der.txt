juonitteluista näyttelyitä työskentelynsä syytteleminen paella salmonella
