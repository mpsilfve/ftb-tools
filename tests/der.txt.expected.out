
"<juonitteluista>"
	"juonittelu" N Pl Ela abc #1
"<näyttelyitä>"
	"näyttely" N Pl Par abc #1
	"näyttelyitä" N Sg Nom abc #2
"<työskentelynsä>"
	"työskentely" N Pl Nom Px3 abc #1
	"työskentely" N Sg Gen Px3 abc #1
	"työskentely" N Sg Nom Px3 abc #1
"<syytteleminen>"
	"syytteleminen" N Lex_Minen Sg Nom abc #1
"<paella>"
	"paella" N Sg Nom abc #1
"<salmonella>"
	"salmonella" N Sg Nom abc #1
